src/main/webapp/app/layouts/navbar/navbar.component.html

```diff
@@ -17,7 +17,7 @@
                 </a>
             </li>
             <!-- jhipster-needle-add-element-to-menu - JHipster will add new menu items here -->
-            <li *ngSwitchCase="true" ngbDropdown class="nav-item dropdown pointer" display="dynamic" routerLinkActive="active" [routerLinkActiveOptions]="{exact: true}">
```

Comment:
Why did you delete this `*ngSwitchCase="true"`?

Reply:
Since *ngSwitchCase="true" hides the content for unregistered users.

Reply:
Ok